core/files/changelog.txt

```diff
@@ -18,6 +18,7 @@
     CH graphs are now identified using CHProfile instead of Weighting, #1670
     removed the 'traversal_mode` request parameter for /route, instead of 'traversal_mode=edge_based_2dir' use edge_based=true
     removed GraphHopper.set/getTraversalMode() methods, #1705
+    removed the 'traversal_mode` request parameter for /route, instead of 'traversal_mode=edge_based_2dir' use edge_based=true
```

Comment:
I think this is a merge conflict and duplicated?

Reply:
ah sorry yes